### ISSUE TITLE
Active links and read link colors not colorblind friendly

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#907]
Carried over from https://github.com/brave/browser-laptop/issues/7842

-------

> From community  https://community.brave.com/t/active-links-and-read-link-colors-not-colorblind-friendly/1787
> >cannot see any difference in the colors used for an active click-able link I have never visited, and one I have. It would be very helpful if the default colors were shifted farther apart (visited links darker than unvisited links by at least 10%). 

-------

> Let's beat Chrome with a color-blind friendly delta.
> 
> If we move towards a turquoise blue, the effect is much more visible in all 3 color blindness types:
> 
> ![image](https://user-images.githubusercontent.com/13509546/27111333-4aa90370-5064-11e7-8fa0-b74587d5b9f3.png)
> 
> I suspect the blue-to-purple standard was chosen to prevent adding extra attention to links unfairly even though you had already visited them. If this becomes a problem, we may need to add color-blindness accessibility mode.
> 
> 
> visible delta with green-blue adjustment shown in 3 major colorblindness tests:
> 
> ![image](https://user-images.githubusercontent.com/13509546/27111391-abd0f2e8-5064-11e7-8a45-dd1a98db11dd.png)
> 
> ![image](https://user-images.githubusercontent.com/13509546/27111412-c068e4c2-5064-11e7-905c-dcee8d0f7524.png)
> 
> 
> ![image](https://user-images.githubusercontent.com/13509546/27111418-cb989b76-5064-11e7-8873-96ec1f7cec9c.png)

-------

> the magic color for visited links shown in the images above is: #00997c
